check_pgactivity

```diff
@@ -3023,7 +3023,7 @@ sub check_commit_ratio {
     };
 
     if ( defined $args{'warning'} ) {
-        my $thresholds_re = qr/(rollbacks|rollback_rate|rollback_ratio)\s*=\s*(\d+)/i;
+        my $thresholds_re = qr/(rollbacks|rollback_rate|rollback_ratio)\s*=\s*([0-9]+\.?[0-9]*)/i;
```

Comment:
Since you can already specify both a number / rate / ratio, I'm not sure that this is required.

Also, decimals don't really make sense with `rollbacks` label.

Reply:
I disagree, number and rate are absolute value whereas ratio is relative. When traffic grows number and rate goes up whereas ratio can remain stable.